### ISSUE TITLE
[Feature request] Bring back arrow feature ￪

### ISSUE THREAD

**Issue by krillin666** [curusarn/resh#170]
The arrow feature is the main feature I use from this amazing project. How can I revert back to the last version ?
Thank you !

**Comment by curusarn** [comment]
Hi, the arrow feature by design behaved very similarly to native bash and zsh arrow up feature.
But it can be tricky to configure your shell to behave properly.

Can you describe what happens when you press arrow up?
Can you describe would you like to happen?

**Comment by krillin666** [comment]
Thank you for taking the time to answer.
Before, on **resh**, if started to write for example `python` on bash, and then I pressed *up* it would replace the current prompt on bash by the latest commands that started with `python` : `python file1.py`, press up again `python file2.py`  and etcetera.
If I now write `python` and press up it will just substitute by the last command I entered on bash as is the default behaviour on bash.
